Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN apt-get update && apt-get install -y \
 	    git \
         libzip-dev \
         unzip \
+        sudo \
```

Comment:
is needed?

Reply:
no, sorry, I was using it but not anymore